packages/teams-js/src/internal/messageObjects.ts

```diff
@@ -46,6 +48,7 @@ export interface SerializedMessageResponse {
   uuidAsString?: string;
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   args?: any[];
+  timestamp?: number;
```

Comment:
why wouldn't this have monotonicTimestamp?

Reply:
This is only for the response that the Hosts sends to Teams JS. For requests, as Teams JS was already using the `timestamp` field we needed to create a new one for backwards compatibiltiy. For responses, we can use `timestamp` instead as it is a completely new field...

If you find it more homogeneous, we can still call it `monotonicTimestamp` for the response to comply with the field name in the request. But it wouldn't be necessary.

Reply:
yes, pls use the same name to minimize confusion